modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1605,19 +1604,15 @@ private CompletableFuture<Void> tableStopFuture(TableImpl table) {
             stopReplicaFutures[p] = stopPartition(replicationGroupId, table);
         }
 
-        CompletableFuture<Void> stopPartitionReplicasFuture = allOf(stopReplicaFutures).orTimeout(10, TimeUnit.SECONDS);
-
-        return stopPartitionReplicasFuture
-                .whenCompleteAsync((res, ex) -> {
-                    Stream.Builder<ManuallyCloseable> stopping = Stream.builder();
-
-                    stopping.add(internalTable.storage());
-                    stopping.add(internalTable.txStateStorage());
-                    stopping.add(internalTable);
-
+        return allOf(stopReplicaFutures)
```

Comment:
Can missing `.orTimeout(10, TimeUnit.SECONDS)` cause freezes?

Reply:
It shouldn't because we do `get(30, TimeUnit.SECONDS)` on the resulting future outside